applications/RomApplication/python_scripts/rom_manager.py

```diff
@@ -166,8 +198,10 @@ def Test(self, mu_test=[None]):
 
 
 
-    def RunFOM(self, mu_run=[None]):
-        self.__LaunchRunFOM(mu_run)
+    def RunFOM(self, mu_run=[None], snapshots_matrix_name=None):
```

Comment:
Let us not store the numpy array with the solution if `snapshots_matrix_name=None`